test/distributed/test_functional_api.py

```diff
@@ -384,23 +401,21 @@ def test_all_reduce(self):
         self.assertIsNone(x.grad)
 
 
-class TestMakeFx(TestCase):
+@skipIfHpu
+class TestMakeFx(MultiThreadedTestCase):
+    @property
+    def world_size(self):
+        return 2
+
     def setUp(self):
-        # make_fx is not thread-safe due to patching nd mutating global states
-        # so create a fake_pg.
-        self.rank = 0
-        self.world_size = 2
-        store = FakeStore()
-        dist.init_process_group(
-            backend="fake",
```

Comment:
This test is meant to run with a single process using fake process group. I don't think it falls in the scope of this PR.

Reply:
You are right, this was unnecessarily changed

Reverted back to the old test in: 7afa48037bfcdc0f2f8837cebcaefcfbf633de89

Thanks